doc/source/myrocks/zenfs.rst

```diff
@@ -30,7 +30,7 @@ The following procedure installs Percona Server for MySQL and then configures
 
 .. note::
 
-   The ``<block_device_name>`` can have a short name designation which is the ``<short_block_device_name>``. For the purposes of this document, the ``block_device_name`` is ``/dev/nvme0n2`` and the short name is ``nvme0n2``. 
+   The ``<block_device_name>`` can have a short name designation which is the ``<short_block_device_name>``. For example, the ``<block_device_name>`` could be ``/dev/nvme0n2`` and the short name would result in ``nvme0n2``.
```

Comment:
When providing an example, it's important to be as simple, and clear, and use an active voice. 
The use of "could be" and "would result in" may cause a reader to disconnect from the sentence, especially since we start the sentence by saying "For example,". 
Maybe something like:
For example, if the <block_device_name> is ``/dev/nvme0n2`` remove the ``/dev/`` portion and the <short_block_device_name>  is ``nvme0n2``.